src/pages/studyView/table/ClinicalEventTypeCountTable.tsx

```diff
@@ -0,0 +1,412 @@
+import { ClinicalEventTypeCount } from 'cbioportal-ts-api-client/dist/generated/CBioPortalAPIInternal';
```

Comment:
Adding this Table seems to be redundant... I need to refactor MultiSelectionTable.tsx to be more generic